Products/ZenModel/ZenPack.py

```diff
@@ -1382,37 +1383,53 @@ def installServiceDefinitions(self, serviceDefs, servicePaths):
         if isinstance(servicePaths, basestring):
             servicePaths = [servicePaths]
 
+        serviceDefs = [json.loads(sd) for sd in serviceDefs]
+        for sd, sp in zip(serviceDefs, servicePaths):
+            sd["parentServicePath"] = sp
+
+        for sd in serviceDefs:
+            sd["Services"] = []
+            sp = sd["parentServicePath"]
+            sp = "" if sp == "/" else sp
+            paths = [sp + "/=" + sd["Name"]]
+            for tag in sd["Tags"]:
+                paths.append(sp + "/" + tag)
```

Comment:
I don't think this is sufficient.  We should cover the name matching ('=foo') and parent ('..') functionality. See https://github.com/zenoss/zenoss-prodbin/blob/develop/Products/ZenUtils/controlplane/servicetree.py#L86-L94